server/test/api/inventory.js

```diff
@@ -0,0 +1,73 @@
+/* jshint expr: true */
+const chai = require('chai');
+const expect = chai.expect;
+
+const helpers = require('./helpers');
+const uuid    = require('node-uuid');
+helpers.configure(chai);
+
+describe('(/inventory) The inventory HTTP API', function () {
+
+  // Logs in before each test
+  const agent = chai.request.agent(helpers.baseUrl);
+  before(helpers.login(agent));
+
+  // totals inventory
+  let inventoryCount = 0;
+
+  // inventory list items
+  let metadata = {
+    uuid : uuid.v4(),
+    code : '1000004', // code must be unique
+    text : 'Inventory Article for Test',
+    price : 5,
+    group_uuid : '1410dfe0-b478-11e5-b297-023919d3d5b0',
+    unit_id : 1,
+    type_id : 1,
+    consumable : 0
+  };
+
+  let metadataUpdate = {
+    uuid : metadata.uuid,
+    code : '1000004', // code must be unique
+    text : '[Update] Inventory Article for Test',
+    price : 10,
+    group_uuid : '1410dfe0-b478-11e5-b297-023919d3d5b0',
+    unit_id : 1,
+    type_id : 1,
+    consumable : 0
+  };
+
+  it('GET /inventory/metadata returns the list of inventory metadata', () => {
+    return agent.get('/inventory/metadata')
+      .then((res) => {
```

Comment:
You should make sure this returns `200 OK`.

Why do you need `inventoryCount`?  Is this for a future test?